group/edwards25519/curve.go

```diff
@@ -52,7 +52,7 @@ func (c *Curve) NewKey(stream cipher.Stream) kyber.Scalar {
 	random.Bytes(buffer[:], stream)
 	scalar := sha512.Sum512(buffer[:])
 	scalar[0] &= 0xf8
-	scalar[31] &= 0x7f
```

Comment:
why was this change? I don't think it's copied from the other PR because I only copied the files in the share folder.